Makefile

```diff
@@ -6,3 +6,8 @@ help:
 
 release:
 	python setup.py register sdist upload
+
+test:
+	@coverage run test_project/manage.py test
+	@flake8 .
+	@DJANGO_SETTINGS_MODULE=test_project.settings coverage report
```

Comment:
I don't think you need `DJANGO_SETTINGS_MODULE=test_project.settings` here and I would group the two commands about coverage together:
```
	@coverage run test_project/manage.py test
	@coverage report
	@flake8 .
```